### PR TITLE
Add AePlainInput component

### PR DESCRIPTION
**Pull request by etharner**
Use this component instead of ae-text in contenteditable mode.

### CODE REVIEW

**Comment by sadiqevani** [comment]
Hey @etharner 

Can you give me an overview to what where the reasons you think having a `plain-input` component would be better than `contenteditable` property? 

In the standard design guide there are not plain inputs, and the only usage of the plain input is in the `card` component.

from what I know `contenteditable` behaves the same as if its an input when its enabled, and also provides less hassle and less components.

**Comment by etharner** [comment]
Hey @sadiqevani
As I see:
  1) you use contenteditable not only in ae-card, but in ae-backdrop and ae-flip.
  2) it behave not quite as enabled input, because it allows to use many shortcuts, which can change text pretty much (for example, ctrl-b, ctrl-i for bold, italic, etc.). So you need to disable it somehow. It's overkill for the component that needs to be simple.                                
  3) you can't receive value from your ae-text component (it's not accessible with v-model and you are not implementing input event handling).

File: config/styleguidist.config.js [hunk 1]
```diff
@@ -23,11 +23,27 @@ module.exports = {
    */
   title: require('../package.json').name + ' \n ' + 'Version: ' + require('../package.json').version,
 
-  /**
-   * Components folder
-   * TODO: later remove '-' for folder recognition
-   */
-  components: path.resolve(__dirname, '../src/components/**/*-*.vue'),
+  components: [
```
Comment (sadiqevani) [inline config/styleguidist.config.js]: This is a bit cumbersome to manually add components, can you revert back to the original code?

File: src/components/index.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import AeListItem from './ae-list-item';
 import AeQRCode from './ae-qrcode';
 import AeText from './ae-text';
 import AeToolbar from './ae-toolbar';
+import AePlainInput from './aePlainInput';
```
Comment (sadiqevani) [inline src/components/index.js]: Rename the folder / files to: `ae-input-plain`

File: src/components/aePlainInput/aePlainInput.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    placeholder: String,
+    value: undefined,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/placeholders/typography';
+@import '../../styles/variables/colors';
+@import '../../styles/variables/animations';
```
Comment (sadiqevani) [inline src/components/aePlainInput/aePlainInput.vue]: Use this import for variables:

```
@import '../../styles/globals';
```

File: src/styles/placeholders/typography.scss [hunk 4]
```diff
@@ -4,6 +4,8 @@
 // *****************************
 //------------------------------
 
+@import '../variables/typography.scss';
```
Comment (sadiqevani) [inline src/styles/placeholders/typography.scss]: There is no need to add this, all of the needed global variables are found in:

```
@import '../../styles/globals';
```

File: src/components/aePlainInput/index.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import AePlainInput from './aePlainInput.vue';
```
Comment (sadiqevani) [inline src/components/aePlainInput/index.js]: The `index.js` file should look like this:

```
import AeBadge from './ae-badge.vue';

export const install = (Vue) => {
  Vue.component('ae-badge', AeBadge);
};

export default AeBadge;
```

**Comment by etharner** [comment]
@sadiqevani I've changed stuff. Ready to merge?

**Comment by sadiqevani** [comment]
@etharner 

Here is all the steps that you need to update your component for me to merge it:

- [ ] change the name to: `ae-input-plain`(this means everything that has that name-space needs to change, the css classes and so on)
- [ ] add a new line after: `@extend %face-sans-base;`
- [ ] Revert back to the changes of `ae-text` in case you want to add or remove something, make a new pull-requests for `ae-text`

**Pull request merged**